.travis.yml

```diff
@@ -43,9 +43,9 @@ matrix:
         - os: linux
           compiler: clang-3.6
           env: CONFIG_OPTS="enable-asan"
-        - os: linux
-          compiler: clang-3.6
-          env: CONFIG_OPTS="enable-msan"
+#        - os: linux
+#          compiler: clang-3.6
+#          env: CONFIG_OPTS="enable-msan"
```

Comment:
As an experiment, have you tried with just `clang`?  Maybe we should have the script part output the compiler version, as I guess that changes over time for the `gcc` and `clang` choices...